### DISCUSSION TITLE
Blog – Multiple authors

### DISCUSSION THREAD

**Discussion by jpeterson** [facebook/docusaurus#4527] [Ideas]
Hey everyone 👋🏼 

Just diving into Docusaurus here – really awesome project ❤️ 

Has there been any discussion around supporting multiple authors for blog posts? The current header options don't seem to allow this, but I'd be interested in ways to achieve it! 

An alternative option in my case is to use something like "The dev team" as an author in order to attribute posts to more than one person.

Thanks!

**Comment by slorber** [comment]
No discussion but I think this is something we could support.

Note we have "blog multi-instance" in case you want  multiple blogs on the same site, but it's not exactly the same 😅

**Reply by jpeterson** [reply]
Thanks for the reply @slorber! This would be a great feature to help attribute posts with co-authors.  

I did note the multi-instance setup (which is cool in its own right!), but yeah not exactly what I'm after 😄

**Reply by slorber** [reply]
This is likely to be implemented as part of https://github.com/facebook/docusaurus/issues/5363

**Reply by slorber** [reply]
Merged in https://github.com/facebook/docusaurus/pull/5396, released soon
